recipes/drwav/all/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
Where did you obtain this cmake file? I see no official there, have you tried to open a PR to owner?

Reply:
I wrote the CMakeLists.txt myself (or more accurately: I copied it from another recipe and adapted it). Since the library is officially single-file header-only, I doubt they would accept a CMakeLists.txt but I have not asked so far.